### PR TITLE
Template Updated

### PR DESCRIPTION
**Pull request by k2fx**
Created a new mobile-friendly layout

### CODE REVIEW

**Comment by umputun** [comment]
This PR seems to brake feed.tmpl template
o7cmf9mrw3-20191018-145210

File: app/webapp/templates/feed.tmpl [hunk 1]
```diff
@@ -2,56 +2,179 @@
...
+            <span class="ump-feed-master-info">{{.Description}} <a href="{{.Link}}" class="ump-feed-master-header-link">«{{.Name}}»</a></span>
+        </div>
+    </div>
+    <div class="ump-feed-master-header__meta">
+        {{len .Items}} feeds, {{.LastUpdate.Format "02 Jan 2006 15:04:05 MST"}}
+    </div>
+</header>
+
+<main class="ump-feed-master">
+
```
Comment (svacad) [inline app/webapp/templates/feed.tmpl]: missing {{range .Items}}

Comment (k2fx) [reply app/webapp/templates/feed.tmpl]: @svacad  thank you!

Comment (k2fx) [reply app/webapp/templates/feed.tmpl]: @umputun I restored missed template part.

**Comment by umputun** [comment]
responsiveness seems to work fine. However, I'm not sure if this black-and-white color scheme is smth you expected. My initial imression was - some css styles are missing

409z2lun6q-20191018-160226

**Comment by k2fx** [comment]
@umputun Nothing missed. I was focused on a new layout. 
If you OK with it then we can style it to something close to эхо москвы website (why not).

**Comment by umputun** [comment]
I'm fine with the layout. However, it has very little to do with эхо москвы website, just an example with some feeds in. This one, for example, has a different set of feeds:

geiv525bu5-20191018-161552

**Comment by k2fx** [comment]
I got it. I'll change the color scheme

**Comment by umputun** [comment]
looking good, thx

**Comment by k2fx** [comment]
NP

**Pull request merged**